experimental/PropertyDDS/packages/property-properties/src/properties/baseProperty.js

```diff
@@ -1200,23 +1158,20 @@ BaseProperty.prototype._setDirtyTree = function (in_reportToView) {
 };
 
 /**
- * Determines whether a property can be inserted as a child of another property
- * This does NOT validate if the parent can accept the child property, it only validates if
- * the child property can be inserted in the parent.
- * @param {property-properties.BaseProperty} in_targetParent - The parent property
- * @param {string} in_childId - The id of the child property
- * @throws if the property can not be inserted
- * @private
+ * Returns the boolean to determine whether a property can be inserted to another property
+ * @return {boolean} True if the property can be inserted. False if the property is already inserted
  */
-BaseProperty.prototype._validateInsertIn = function (in_targetParent, in_childId) {
+BaseProperty.prototype._canInsert = function (in_targetParent) {
```

Comment:
Missing doc for `in_targetParent`

Reply:
Perhaps we should rename it to `_canInsertTo`, it seems like this what it tries to do.